client/my-sites/media-library/style.scss

```diff
@@ -115,3 +115,27 @@
 .media-library__list-item.is-small .media-library__list-item-file-extension {
 	display: none;
 }
+
+.media-library__videopress-nudge-container {
+	display: block;
+	overflow-y: auto;
+	position: absolute;
```

Comment:
Is the absolute positioning really necessary here? Seems like we're just adding some padding/margin to the top. It's best to avoid absolute position unless there's some reason that we need it here.